src/reconstruction/StandardModel.jl

```diff
@@ -103,7 +106,7 @@ macro add_reactions!(model::Symbol, ex::Expr)
         end
         push!(all_reactions.args, :(add_reaction!($model, r)))
     end
-    return all_reactions
+    nothing
```

Comment:
this is unnecessary (for-cycles implicitly return nothing)

Reply:
fixed